janitor/io.py

```diff
@@ -1,34 +1,49 @@
 import os
 from glob import glob
+from typing import Iterable, Union
 
 import pandas as pd
 
+from .utils import deprecated_alias
 
-def read_csvs(filespath: str, seperate_df: bool = False, **kwargs):
+
+@deprecated_alias(seperate_df="separate_df", filespath="files_path")
```

Comment:
Great catch here on the spelling! I wish there was an automatic way to catch these.

Reply:
Sure thing! I find code spell checkers hit or miss with variable names, but I wouldn't be surprised if there is something decent out there already.